dev.sh

```
@@ -95,40 +99,40 @@ stop_mcp_server() {
     log_info "Checking Docker Compose services..."
-    
+
     # Check if docker-compose.yml exists
     if [ ! -f "docker-compose.yml" ]; then
         log_error "docker-compose.yml not found in current directory"
         exit 1
     fi
-    
+
     # Check if services are running
     local qdrant_status=$(docker compose ps -q qdrant 2>/dev/null | wc -l)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)